compose/docker-compose.yml

```diff
@@ -51,7 +51,6 @@ services:
       context: ./proxy
       args:
         APP_HOST: web
-    image: openproject/proxy:${TAG:-14-slim}
```

Comment:
🔴 Having an image tag prevents building an temporary image every time on `docker compose up`.

Also, having an image tag should not break `docker compose pull`. Yet, the correct command to update a stack with a local build context is `docker compose build --pull`. We should update the README with that.

Reply:
The problem is that it does indeed break in some versions. I'd rather accept you rebuild the basic image than it failing on these versions.

Reply:
but `docker compose build --pull` does not break, does it?

Reply:
@oliverguenther @Kharonus I propose #91 instead of this PR to fix the issue in the future. I have also added a troubleshooting section so that this error may be found via searches in the future.